### PR TITLE
Task/asv 1776 dark theme reviews

### PR DESCRIPTION
**Pull request by paquiquinho**
**What does this PR do?**

This PR adds dark theme to the reviews view. Since this view is still using widgets, themes had to be duplicated at this point for it to work. Also had to inject dark theme information further down the code. This means that some methods are at this point being passed false for dark theme regardless of what is set. These places have no influence on the theme at this time though, as their being used to reference attributes that are the same on both. A new task should be opened to take care of this in the future;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] StoreTheme.java

**How should this be manually tested?**

Make sure the theme applies correctly on both white and dark;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1776](https://aptoide.atlassian.net/browse/ASV-1776)

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

### CODE REVIEW

File: app/src/main/res/layout/comment_layout.xml [hunk 1]
```diff
@@ -42,6 +43,7 @@
         android:textSize="@dimen/text_size_extra_small"
         android:textStyle="bold"
         tools:text="Mary Pollaki"
+        style="@style/Aptoide.TextView.Medium.XXS.Theme"
```
Comment (phillipsgoncalves) [inline app/src/main/res/layout/comment_layout.xml]: should not use these theme style.

File: app/src/main/res/layout/fragment_rate_and_reviews.xml [hunk 2]
```diff
@@ -18,8 +18,8 @@
       xmlns:tools="http://schemas.android.com/tools"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
-      android:background="@color/white"
       android:orientation="vertical"
+      android:background="?attr/widgetBackgroundColorPrimary"
```
Comment (phillipsgoncalves) [inline app/src/main/res/layout/fragment_rate_and_reviews.xml]: codestyle

File: app/src/main/res/values-v17/styles.xml [hunk 3]
```diff
@@ -22,7 +22,6 @@
     <item name="android:indeterminate">false</item>
     <item name="android:max">100</item>
     <item name="android:progress">0</item>
-    <item name="android:progressDrawable">@drawable/dark_progress_bar</item>
+    <item name="android:progressDrawable">?attr/ratingBarDrawable</item>
```
Comment (phillipsgoncalves) [inline app/src/main/res/values-v17/styles.xml]: why did you add this and the ones in value-v23 file in separate files?

File: app/src/main/res/values/colors.xml [hunk 4]
```diff
@@ -69,4 +69,8 @@
   <color name="catappult_pink">#ff057b</color>
   <color name="appc_orange">#fd6363</color>
 
+  <!--Widgets-->
```
Comment (phillipsgoncalves) [inline app/src/main/res/values/colors.xml]: why didnt you use the attribute in rigth away where you want to use the color?

File: app/src/main/java/cm/aptoide/pt/reviews/RateAndReviewsFragment.java [hunk 5]
```diff
@@ -197,8 +197,9 @@ public String getAppName() {
   @Override public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
     super.onViewCreated(view, savedInstanceState);
     if (storeTheme != null) {
-      ThemeUtils.setStatusBarThemeColor(getActivity(), storeTheme);
-      ThemeUtils.setStoreTheme(getActivity(), storeTheme);
+      ThemeUtils.setStatusBarThemeColor(getActivity(), null);
+      ThemeUtils.setStoreTheme(getActivity(), null);
+      view.setBackgroundColor(getResources().getColor(R.color.black));
```
[the remark this answers is not in the document]
Comment (paquiquinho) [reply app/src/main/java/cm/aptoide/pt/reviews/RateAndReviewsFragment.java]: This was a leftover from previous tests, it is not needed at all;

File: app/src/main/res/values/colors.xml (hunk 4 above)
Comment (paquiquinho) [reply app/src/main/res/values/colors.xml]: This was a leftover from a previous test. Removed;
